types/react-relay/relay-hooks/EntryPointTypes.d.ts

```diff
@@ -157,7 +157,7 @@ export interface PreloadProps<
     TPreloadParams extends {},
     TPreloadedQueries extends Record<string, OperationType>,
     TPreloadedEntryPoints extends Record<string, EntryPoint<any, any> | undefined>,
-    TExtraProps extends {} | null
+    TExtraProps extends {} | null,
```

Comment:
is this stylistic change something introduced by our formatter?

Reply:
Introduced by prettier, see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.prettierrc.json#L5

 * `npm run prettier -- --write types/relay-runtime/**/*.ts`
 * `npm run prettier -- --write types/react-relay/**/*.ts`